README.md

```diff
@@ -232,3 +232,78 @@ package with features:
       -- code of the function
   end)
   ```
+
+## Tarantool 3.0 role
+
+`roles.expirationd` is a Tarantool 3.0 role for the expirationd
+package with features:
+
+* You can configure the expirationd role with `cfg` entry (check example).
+  cluster configuration allows to set the same parameters as
+  in [expirationd.cfg()](https://tarantool.github.io/expirationd/#cfg)
+* You can use persistent functions (i.e. created by `box.schema.func.create`).
+  When configuring, role tries firstly get function from global namespace
+  (`_G`) and if function was not found then role tries search in `box.func` for
+  function with the same name.
+  If some functions from config are missing, expirationd will wait for their
+  creation and start tasks when all of them are found. You can check alerts
+  to see what functions are missing.
+* The role stops all expirationd tasks on an instance on the role termination.
+* The role can automatically start or kill old tasks from the role
+  configuration.
+
+  ```yaml
+  roles: [roles.expirationd]
+    roles_cfg:
+      roles.expirationd:
+        cfg:
+          metrics: true
+        task_name1:
+          space: users
+          is_expired: is_expired_func_name
+          is_master_only: true
+          options:
+            args:
+            - any
```

Comment:
I think it's more common for us to stick to the following indentation format:
```yaml
list:
  - item1
  - item2
```

Reply:
changed